package.json

```
@@ -88,10 +88,11 @@
     "@mui/lab": "^5.0.0-alpha.63",
     "@mui/material": "^5.4.0",
     "@mui/styles": "^5.3.0",
+    "@mui/x-date-pickers": "^5.0.0",
     "@tanstack/react-query": "^4.1.0",
     "@visx/visx": "2.9.0",
     "axios": "^0.25.0",
-    "centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#develop",
+    "centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#fix-contrast-ratio",
```

I assume that it should point to develop after merge :
```suggestion
    "centreon-frontend": "git+https://centreon@github.com/centreon/centreon-frontend.git#develop",
```

---

Yup. I did it in order to check the Lighthouse report generated by the CI